lib/seed_utils.rb

```diff
@@ -3,6 +3,41 @@
 module SeedUtils
   module_function
 
+  def seed_tables_with_group(group, user, options = {})
+    group_options = Group.group_config_options(group)
+    sheets = []
+    group_options[:types].each do |type|
+      sheets << {
+        klass: type,
+        skip_lines: 0
+      }
+    end
+
+    file_options = { sheets: sheets }
+    csv_name = "#{group}.xlsx"
+    csv_path = 'sample_csvs'
+
+    load_table(Group, user, file_options.reverse_merge(options), csv_name)
+
+    uf = ActionDispatch::Http::UploadedFile.new(
```

Comment:
this should be pulled out into its own method
`def upload_file(name, content_type)`